content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -231,12 +240,19 @@ status:
   qosClass: Burstable
 ```
 
-Delete your Pod:
+## Retrieve the QoS class for a Pod
 
-```shell
-kubectl delete pod qos-demo-4 --namespace=qos-example
+Rather than see all the fields, you can view just the field you need:
```

Comment:
I don't know that this sentence is relevant since this is a separate section. I think you can replace with "Check the `status.qosClass` field:"